blog/2024-12-13-year-in-review-2024.md

```diff
@@ -0,0 +1,82 @@
+
+## Expanding Our Reach
+
+We reached a growing audience this year and continued our work with a range of communities, including journalists and media professionals, GLAM (galleries, libraries, archives, and museums), Wikipedians and Wikimedia contributors, scientists and researchers across a range of disciplines, data analysts, and educators. More and more people are joining the community and are enthusiastic about learning, using, and teaching the tool. As librarian Thomas Guignard shared earlier this year, 
+> Teaching OpenRefine is the absolute best – people feel so much joy that it exists.
+
+### Highlighting Our Usage
+
+OpenRefine was downloaded on average 15,500 times per month and received over 800 academic citations in our last full year of operation (2023). Since the start of 2024, activity on our GitHub has been bustling, with 33 active GitHub contributors, 241 issues created and 192 closed, and 223 PRs merged. Our average number of active GitHub contributors per month has [remained steady for the past two years](/usage#contribution-statistics). 
+
+Additionally, we've seen a burst of activity on our forum, with 271 new users signed up in the past year, for a new total of 637 forum users. This year, 364 topics were created for a total of 1,900 messages. Our 2024 survey reflected this engagement, with 226 responses (up from 207 in 2023) across skill levels, geography, and tenure with the tool. Respondents also reported using the tool in over 30 languages. 
+
+We are deeply grateful for our active community on both GitHub and the forum, and believe our tool is strengthened by the range of perspectives we've engaged this year. 
+
+
+### Welcoming Wikimedia Sweden
+
+This year, we enthusiastically embraced a new partnership with [Wikimedia Sweden (WMSE)](https://wikimedia.se/), who joined us to make improvements to our Wikimedia Commons integration. This partnership took shape thanks to the Wikimedia-OpenRefine training and sustainability project. Through this partnership, WMSE developer ramped up on OpenRefine code and tackled strategic bug fixes and other enhancements to address feedback we had received since the integration was first released. 
```

Comment:
```suggestion
This year, we enthusiastically embraced a new partnership with [Wikimedia Sweden (WMSE)](https://wikimedia.se/), who joined us to make improvements to our Wikimedia Commons integration. This partnership took shape thanks to the Wikimedia-OpenRefine training and sustainability project. Through this partnership, WMSE developers ramped up on OpenRefine code and tackled strategic bug fixes and other enhancements to address feedback we had received since the integration was first released. 
```
If they were not multiple, I would suggest "a" in front of WMSE.